pkg/middlewares/loopstop/loop_stop_test.go

```diff
@@ -0,0 +1,30 @@
+package loopstop
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestServeHTTP(t *testing.T) {
+	req, err := http.NewRequest(http.MethodGet, "", nil)
+	require.NoError(t, err)
+
+	m, err := NewLoopStop("myRouter", http.HandlerFunc(func(_ http.ResponseWriter, _ *http.Request) {}))
```

Comment:
I think it could be great to assert that the next router is called and not called when we have a recursion.